src/wizard/android/index.md

```diff
@@ -40,6 +40,9 @@ Add your DSN to the manifest file.
 ```xml {filename:AndroidManifest.xml}
 <application>
     <meta-data android:name="io.sentry.dsn" android:value="___PUBLIC_DSN___" />
+    <!-- Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
+       We recommend adjusting this value in production. -->
+    <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
```

Comment:
there's already a section down this page, called Performance Monitoring, should we repeat it?

Reply:
we can link down this page to the automatic instrumentation that requires adding an integration, similar to RN

Reply:
Discussed on a call but to have it here: We want to experiment having this opt-in on the first copy-paste users do.